pegasus/sites.v3/code.org/public/teacher-dashboard/section_manage.haml

```diff
@@ -172,3 +172,7 @@ content-type: text/ng-template
 
   #login-type-react{'ng-init' => '$emit("login-type-react-rendered");'}
   #share-setting-react{'ng-init' => '$emit("share-setting-react-rendered");'}
+
+  %br/
+  %div{id: 'privacy_link'}
+    =I18n.t('dashboard_privacy_document_link')
```

Comment:
Looks like the string is: `<a href="/privacy/student-privacy", target="_blank">Click here</a> for our privacy document that can be shared or printed for your students.`

I remember @Bjvanminnen saying we don't love adding html into strings, although there seem to be lots of html in our string file. Should be be substituting in variables here?

Reply:
Yeah, that's not good. I'm also a little surprised you can just embed a string like that in haml and have it work